src/components/stats.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import './stats.css'
+import Vest from '../icons/gear_types/vest.png'
+import BreakdownItem from "./breakdownItem";
+
+function Stats() {
+    return (
+        <div className="statsGrid">
+            <div className="grid sectionSelectionGrid">
+                <img className="dsIco" src={Vest} onClick ="" alt="Weapon Talents"/>
```

Comment:
Can this class name have a better name? idk what `dsIco `means.

Reply:
Changed the class name to something more readable in the most recent commit